packages/block-library/src/navigation/index.php

```diff
@@ -65,68 +65,84 @@ function block_core_navigation_sort_menu_items_by_parent_id( $menu_items ) {
 
 		return $menu_items_by_parent_id;
 	}
-}
 
+	/**
+	 * Gets the inner blocks for the navigation block from the unstable location attribute.
+	 *
+	 * @param array $attributes The block attributes.
+	 * @return WP_Block_List Returns the inner blocks for the navigation block.
+	 */
+	function block_core_navigation_get_inner_blocks_from_unstable_location( $attributes ) {
+		$menu_items = block_core_navigation_get_menu_items_at_location( $attributes['__unstableLocation'] );
+		if ( empty( $menu_items ) ) {
+			return '';
+		}
+
+		$menu_items_by_parent_id = block_core_navigation_sort_menu_items_by_parent_id( $menu_items );
+		$parsed_blocks           = block_core_navigation_parse_blocks_from_menu_items( $menu_items_by_parent_id[0], $menu_items_by_parent_id );
```

Comment:
I'm seeing a warning `Deprecated: Function gutenberg_block_core_navigation_parse_blocks_from_menu_items is deprecated since version 6.3.0! Use WP_Navigation_Fallback::parse_blocks_from_menu_items instead. in /var/www/html/wp-includes/functions.php on line 6031` on trunk when using a hybrid theme with some legacy Navigation blocks in it. Would it be possible to use `WP_Navigation_Fallback::parse_blocks_from_menu_items` here?